backend/api/views.py

```diff
@@ -0,0 +1,64 @@
+from django.shortcuts import render
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from .models import User
+
+class LoginView(APIView):
+    
+    def post(self,request):
+
+        if 'user_name' in request.data:
+            user = User.objects.filter(username=request.data.get('user_name')).first()
+        else:
+            user = User.objects.filter(email=request.data.get('email')).first()
+
+        if not user or not user.check_password(request.data.get('password')):
+            return Response({
+                'token':'not ok',
```

Comment:
We should return a 4** status code indicating an error

Reply:
Error will be handled 4** rest-framework status code @nevzat-ersoy-2015400090